fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -117,6 +117,13 @@ public class Config extends ConfigBase {
     @ConfField
     public static String audit_log_delete_age = "30d";
 
+    @ConfField(mutable = true)
+    public static long slow_lock_threshold_ms = 3000L;
```

Comment:
The two configuration can be unified to one configuration.

Reply:
Although currently they have the same value, but they have different usages, for example, when i was doing some tests, i want to set `slow_lock_log_every_ms` to a relative low value, say `100ms`, but keep `slow_lock_threshold_ms` unchanged.